scripts/generate-transcript.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+const moment = require("moment");
+const generateTranscript = require("discord-transcript-generator");
+const { TOKEN, ID, ISSUE_TITLE } = process.env;
+
+/**
+ * Extract a date string from an automatically generated TSC Meeting issue title.
+ * Issue title is formatted as "TSC meeting DD-MMMM-YYYY".
+ * https://github.com/eslint/eslint-github-bot/blob/master/src/plugins/recurring-issues/index.js#L202
+ * @param {string} issueTitle The issue title from which to extract the date string.
+ * @returns {string} The extracted date string.
+ */
+function extractDateString(issueTitle) {
+    const [, dateString = null] = /(\d\d-.+-\d\d\d\d)$/u.exec(issueTitle) || [];
+
+    if (!dateString) {
+        throw new Error("Can't extract date from issue title. Expecting format \"TSC meeting DD-MMMM-YYYY\".");
+    }
+
+    return dateString;
+}
+
+/**
+ * Formats a date string to MM/DD/YYYY.
+ * @param {string} dateString The date string to format.
+ * @returns {string} The formatted date string.
+ */
+function formatDate(dateString) {
+    return moment(new Date(dateString)).format("MM/DD/YYYY");
```

Comment:
Rather than trusting the `Date` constructor to guess correctly, I usually use moment’s parse format argument to be specific:

```suggestion
    return moment(dateString, “DD-MMMM-YYYY”).format(“MM/DD/YYYY”);
```